src/integrationTest/resources/tests/arm/properties/live/media-record.properties

```diff
@@ -0,0 +1,35 @@
+#OBJECT_TYPE
+#CASE_NUMBERS
+#FILE_TYPE
+#HEARING_DATE
+#CHECKSUM
+#TRANSCRIPT_REQUEST
+#TRANSCRIPT_TYPE
+#TRANSCRIPT_URGENCY
+#COMMENTS
+#PLACEHOLDER_DATE
+#CREATED_DATE_TIME
+#OBJECT_ID
+#PARENT_ID
+#CHANNEL
+#MAX_CHANNELS
+#UPLOADED_BY
+#START_DATE_TIME
+#END_DATE_TIME_KEY
```

Comment:
Should this be `#END_DATE_TIME`?

Reply:
This line is commented out just to show all the attributes available

Reply:
It's ok - I seen you corrected this line yesterday